code/game/objects/effects/misc.dm

```diff
@@ -129,3 +129,15 @@
 	. = ..()
 	if(!ismob(loc))
 		return INITIALIZE_HINT_QDEL
+
+/obj/effect/snap_rt
+	icon = 'icons/effects/filters.dmi'
+	icon_state = "nothing"
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	plane = FLOOR_PLANE
+
+/obj/effect/snap_rt/New(_loc, id)
```

Comment:
is there a reason this is using `New` over `Initialize`?

Reply:
I was going to make this a datum then realised that doesnt work